content/_data/upgrades/2-190-1.adoc

```diff
@@ -1 +1,10 @@
-No notable changes requiring upgrade notes.
+==== Jenkins no longer creates symbolic links inside project or build directories
+
+https://issues.jenkins-ci.org/browse/JENKINS-37862[JENKINS-37862]
+
+There were a lot of bugs and limitations with use of symbolic links for build directories, especially on platforms like Windows.
+This functionality was not widely used.
+
+Starting from Jenkins LTS 2.190.1 Jenkins no longer creates symbolic links inside project or build directories.
```

Comment:
> Starting from Jenkins LTS 2.190.1

This might be considered redundant, given that this is in the 2.190.1 upgrade guide section. Unsure.